demisto_sdk/commands/common/native_image.py

```diff
@@ -136,25 +154,29 @@ def __get_ignored_native_images(self):
         return []
 
     def get_supported_native_image_versions(
-        self, get_raw_version: bool = False
+        self, get_raw_version: bool = False, ignore_latest: bool = True
     ) -> List[str]:
         """
-        Get the native-images that the integration/script supports. Disregards native-images that are supported which
-        should be ignored.
+        Get the native-images that the integration/script supports. Disregards native-images that should be ignored.
 
         Args:
             get_raw_version (bool): whether to extract the raw server version from the native image name, for example:
-              'native:8.2' will become '8.2' for each one of the native-images that are supported.
+                                    'native:8.2' will become '8.2' for each one of the native-images that are supported.
+            ignore_latest (bool): whether to ignore the latest native image.
```

Comment:
change [this](https://github.com/demisto/demisto-sdk/pull/2613/files#r1058998727) here too